components/posts/recommandedPosts.js

```diff
@@ -0,0 +1,71 @@
+import Link from "next/link";
+import Image from "next/image";
+
+export default function RecommandedPosts({ Data }) {
+  const [categoryPosts, categoryName] = Data;
+  return (
+    <>
+      <span className="text-[1.7rem] lg:text-[2rem] xl:text-[1.6rem] font-semibold tracking-wide">
+        {categoryName}
```

Comment:
Keep it as `Recommended for you`. If there is no post, the title should not be displayed.